preview_tasks/alpha-mainnet-preview-task.md

```diff
@@ -40,14 +40,19 @@ We also invite you to visit https://www.golem.network/, where you can find more
 - Send us your wallet address (so we can see real GLMs), the console output of the code execution on the mainnet, and Ray on Golem debug log file.
 - Fill out the [submission form](https://qkjx8blh5hm.typeform.com/to/UlpvzPrD) (wallet, the logs, and meaningful and thought-out feedback)
 
+**Troubleshooting**
+
+Should anything goes wrong check out the [troubleshooting section](https://docs.golem.network/docs/creators/ray/troubleshooting). 
+If this doesn't help please reach out to us on [Discord: `#Ray on Golem` channel](https://discord.gg/golem). 
+Ideally, please also upload the [webserver debug logs](https://docs.golem.network/docs/creators/ray/troubleshooting#ray-on-golem-s-log-files).
+
 ### The documentation
 
 While going over the documentation consider its clarity, purpose, structure & completeness - we will be asking about this at the end.
 The direction we should take with the docs is an important aspect of the Preview Program for us.
 
 After you succeed in running the code on the mainnet please read two new articles. 
```

Comment:
should it be obvious _which_ new articles? it's _not_ obvious even to me --'

Reply:
listed the articles :) good catch